src/components/Dashboard/TodoList.vue

```diff
@@ -561,13 +590,17 @@ export default {
       }
     },
 
-    loadAnnualReport (task) {
+    async loadAnnualReport (task) {
       let date
       const filing = task.task.filing
       if (filing && filing.header && filing.annualReport) {
         filing.annualReport.annualReportDate
           ? date = filing.annualReport.annualReportDate
           : date = filing.annualReport.nextARDate
+
+        const bcolErr = filing.header.paymentErrorType || null
+        const bcolObj = bcolErr && await this.getErrorObj(bcolErr)
```

Comment:
If getObjError() fails (network error) then an exception is generated but is not caught here, so the caller (loadAnnualReport) will exit and all callers upstream until the exception is caught or Javascript exits. Please add `.catch()` or wrap this in `try..catch` block.

Reply:
Never mind. I see you're now catching errors in getErrorObj().